tools/diag/gcollect/src/gcollect/main.go

```diff
@@ -0,0 +1,906 @@
+package main
+
+import (
+  "flag"
+  "fmt"
+  "os"
+  "io"
+  "bufio"
+  "bytes"
+  "os/user"
+  "io/ioutil"
+  "logs"
+  "log"
+  "sort"
+  "strconv"
+  "strings"
+  "utils"
+  "time"
+  "tsar"
+  "path/filepath"
+  "golang.org/x/crypto/ssh"
```

Comment:
do we use this?

Reply:
Yes. "time" is used to parse DateTime. "tsar" is a module introduced to handle tsar logs. "filepath" is used to find absolute path of "output" folder (for other nodes to copy results back to the master.) "ssh" is used to setup ssh connection with other nodes.